src/prefect/environments/execution/base.py

```diff
@@ -37,6 +37,9 @@ class Environment:
     def __init__(self) -> None:
         pass
 
+    def __repr__(self) -> str:
+        return "<{}: {}>".format(type(self).__name__, type(self).__name__)
```

Comment:
Could you update this one similar to `ResultHandler`s? So `<Environment: {}>.format(...)`